src/app/zap-templates/zcl/data-model/chip/operational-credentials-cluster.xml

```diff
@@ -24,6 +24,7 @@ limitations under the License.
     <item fieldId="3" name="FabricID" type="fabric_id"/>
     <item fieldId="4" name="NodeID" type="node_id"/>
     <item fieldId="5" name="Label" type="char_string" length="32"/>
+    <item fieldId="6" name="VidVerificationStatement" type="octet_string" optional="true" length="85"/>
```

Comment:
This matches spec, but generally we are trying to use all-caps for acronyms in spec, so spec should probably be updated to VIDVerificationStatement?

Reply:
Will do in spec